server/operator.go

```diff
@@ -220,16 +229,21 @@ func (lto *TransferLeaderOperator) Do(region *metapb.Region, leader *metapb.Peer
 		return true, nil, nil
 	}
 
-	// If lto.mustSuccess is true, then lto.check should always be true.
-	if lto.mustSuccess {
-		return false, nil, errors.Errorf("transfer leader operator called twice - %v", lto)
+	// If lto.count is greater than 0, then we should check whether it exceeds the lto.maxWaitCount.
+	if lto.count > 0 {
```

Comment:
Yep, later we will move it as config flag.
Then we can check it.
Now we have set it directly, so it will never be zero.